api/sql/schema/750$transfer.transfer.get.sql

```diff
@@ -17,21 +20,27 @@ AS
    
    SELECT 'transfer' AS resultSetName
    
-   SELECT transferId, transferTypeId, acquirerCode, transferIdAcquirer, transferIdLedger, transferIdIssuer, transferIdMerchant, 
-   transferDateTime, localDateTime, t.settlementDate, channelId, channelType, ordererId, merchantId, merchantInvoice, 
-   merchantPort, merchantType, cardId, sourceAccount, destinationAccount, expireTime, expireCount, reversed, retryTime, 
-   retryCount, ledgerTxState, issuerTxState, acquirerTxState, merchantTxState, issuerId, ledgerId, transferCurrency, transferAmount, 
-   acquirerFee, issuerFee, transferFee, [description], pi.port issuerPort, pl.port ledgerPort
+   SELECT t.transferId, t.transferTypeId, t.acquirerCode, t.transferIdAcquirer, t.transferIdLedger, t.transferIdIssuer, t.transferIdMerchant, 
+          t.transferDateTime, t.localDateTime, t.settlementDate issuerSettlementDate, t.channelId, t.channelType, t.ordererId, t.merchantId, t.merchantInvoice, 
+          t.merchantPort, t.merchantType, t.cardId, t.sourceAccount, t.destinationAccount, t.expireTime, t.expireCount, t.reversed, t.retryTime, 
+          t.retryCount, t.ledgerTxState, t.issuerTxState, t.acquirerTxState, t.merchantTxState, t.issuerId, t.ledgerId, t.transferCurrency, t.transferAmount, 
+          t.acquirerFee, t.issuerFee, t.transferFee, t.[description], pi.port issuerPort, pl.port ledgerPort, e.udfDetails udfAcquirer
    FROM  [transfer].[transfer] t
    JOIN  [transfer].[partner] pi on pi.partnerId = t.issuerId
    LEFT JOIN [transfer].[partner] pl on pl.partnerId = t.ledgerId
-   WHERE (@transferIdAcquirer IS NULL OR [transferIdAcquirer] = @transferIdAcquirer)
-        AND (@transferId IS NULL OR [transferId] = @transferId)
+   LEFT JOIN [transfer].[event] e ON e.transferId = t.transferId AND e.source = 'acquirer' AND e.type = 'transfer.push'
+   WHERE (@transferIdAcquirer IS NULL OR t.[transferIdAcquirer] = @transferIdAcquirer)
+        AND (@transferId IS NULL OR t.[transferId] = @transferId)
+        AND (@cardId IS NULL OR t.cardId = @cardId)
+        AND (@localDateTime IS NULL OR t.localDateTime LIKE '%' + @localDateTime)
+        AND (@acquirerCode IS NULL OR t.acquirerCode = @acquirerCode)
 
    SELECT 'transferSplit' AS resultSetName
 
-   SELECT splitId, ts.transferId, conditionId, splitNameId, debit, credit, amount, ts.[description], tag, debitActorId, creditActorId, debitItemId, creditItemId, [state], [transferIdPayment]
+   SELECT ts.splitId, ts.transferId, ts.conditionId, ts.splitNameId, ts.debit, ts.credit, ts.amount, 
+          ts.[description], ts.tag, ts.debitActorId, ts.creditActorId, ts.debitItemId, ts.creditItemId, 
+          ts.[state], ts.[transferIdPayment]
    FROM [transfer].[split] ts 
    JOIN [transfer].[transfer] tt ON ts.transferId = tt.transferId
-   WHERE (@transferIdAcquirer IS NULL OR [transferIdAcquirer] = @transferIdAcquirer)
+   WHERE (@transferIdAcquirer IS NULL OR tt.[transferIdAcquirer] = @transferIdAcquirer)
```

Comment:
remove the JOIN and use ts.transferId = transferId acquired from the SELECT TOP 1 query